packages/icons/icons.json

```
@@ -4051,5 +4051,19 @@
         "tags": "microphone, mic, speaker",
         "group": "media",
         "codepoint": 62275
+    },
+    {
+        "displayName": "Geotime",
+        "iconName": "geotime",
+        "tags": "geo, time, date, location, spatial",
+        "group": "interface",
+        "codepoint": 62276
+    },
+    {
+        "displayName": "Unlink",
+        "iconName": "link-off",
```

I think it should be called "unlink", and the file should be renamed rather than the icon

```suggestion
        "iconName": "unlink",
```